internal/engine/wazevo/ssa/instructions.go

```diff
@@ -1748,6 +1742,15 @@ func (i *Instruction) AsVSshr(x, amount Value, lane VecLane) *Instruction {
 	return i
 }
 
+// AsScalarToVector initializes this instruction as move scalar to vector instruction with OpcodeScalarToVector.
```

Comment:
could you fix the comment on `OpcodeScalarToVector` to reflect that it needs to zero out the higher bits?